src/redturtle/voltoplugin/editablefooter/upgrades/configure.zcml

```diff
@@ -0,0 +1,16 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    >
+
+  <genericsetup:upgradeStep
+      title="Add new field in controlpanel"
+      description=""
+      profile="redturtle.voltoplugin.editablefooter:default"
+      source="1000"
+      destination="1001"
+      handler=".upgrades.update_registry"
```

Comment:
You are updating the registry, but I don't see any files added/changed, did you forget in the commit?

Reply:
i added a new field in interfaces.py

Reply:
i don't know (never done) if there is a way to add it manually

Reply:
afaik you need to reload registry